build/android/app/build.gradle

```diff
@@ -63,8 +63,13 @@ task prepareAssets() {
 	copy {
 		from "${projRoot}/builtin" into "${assetsFolder}/builtin"
 	}
-	copy {
+	/*copy {
+		// Minetest shaders not worked with ogles right now
+		// ToDo: fix it!
 		from "${projRoot}/client/shaders" into "${assetsFolder}/client/shaders"
+	}*/
```

Comment:
should be removed?

Reply:
IDK, maybe someday I or someone else will find a way to fix this

Reply:
Should not be removed, but `// TODO: fix Minetest shaders currently not working with OpenGL ES` would IMO be a better comment.